source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -195,7 +197,9 @@ content: |
 
   2. Once you have the required information, update and run the following code
      to generate the new data encryption key:
-
+     
+  .. include:: /includes/substitute-placeholders.rst
+  
```

Comment:
Nit: Having this appear twice in each guide seems like too much to me. Is it safe to assume that if they go through the guide sequentially once is enough? No supporting evidence either way for this so leaving it as a nit.